www-src/app/replicator/change/change_tag_handler.coffee

```diff
@@ -1,15 +1,25 @@
+AndroidCalendarHandler = require "../../lib/android_calendar_handler"
+CozyToAndroidCalendar = require "../transformer/cozy_to_android_calendar"
 log = require('../../lib/persistent_log')
     prefix: "ChangeTagHandler"
     date: true
 
 module.exports = class ChangeTagHandler
```

Comment:
Could you put a comment, on top, explaining this TagHandler, handles only calendar tags (and only deletion).